open-metadata-implementation/server-chassis/server-chassis-spring/SwaggerGeneration.md

```diff
@@ -66,7 +66,7 @@ You can customize the endpoints/paths that are to be described by following the
 
 ## Endpoints produced
 
-The following endpoints are produced in the server chassis. 
+The following endpoints are produced in the ui and server chassis. 
```

Comment:
Not a problem as such, but since the docs here are purely about the server chassis, we may not really need this text?

Reply:
Minor typo still to be fixed

Reply:
fixed now